db/migrate/20181001131632_add_conversion_host_id_to_miq_request_tasks.rb

```diff
@@ -0,0 +1,46 @@
+class AddConversionHostIdToMiqRequestTasks < ActiveRecord::Migration[5.0]
+  class MiqRequestTask < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+    serialize :options, Hash
+    belongs_to :conversion_host, :class_name => "AddConversionHostIdToMiqRequestTasks::ConversionHost"
+  end
+
+  class ConversionHost < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+    belongs_to :resource, :polymorphic => true
```

Comment:
I don't think you can use polymorphic relations with the stub models.  If you do, you'll get something like: `:resource_type => MigrationClass::StubModel` which won't be found in production.

Reply:
Yeah you're right I thought in production it would use the type from the record but appears not

Reply:
Okay, who wants to take on changing this? Or should we write a follow up migration to alter all the broken resources to the "production ready" name?

Reply:
I can, should be able to just edit this migration?

Reply:
If we're going to update the `.where` to `.in_my_region.find_each` anyway

Reply:
I think Jason would say we need a new migration since developers may have already run this one...

Reply:
Okay https://github.com/ManageIQ/manageiq-schema/pull/284 data migration to fix the resource_type